src/commands/show.ts

```diff
@@ -49,7 +49,25 @@ async function showProblemInternal(id: string): Promise<void> {
             return;
         }
 
-        const outDir: string = await selectWorkspaceFolder();
+        let outDir: string = await selectWorkspaceFolder();
+        const outputPath: string = leetCodeConfig.get<string>("outputPath") || "root";
+        switch (outputPath) {
+            case "root": {
+                break;
+            }
+            case "tag": {
+                const { tags } = await leetCodeExecutor.getCompaniesAndTags();
```

Comment:
`Tag` and `Difficulty` is stored in the `LeetCodeNode`. 

See: https://github.com/jdneo/vscode-leetcode/blob/master/src/explorer/LeetCodeNode.ts#L28-L34
https://github.com/jdneo/vscode-leetcode/blob/master/src/commands/show.ts#L15-L19